### ISSUE TITLE
syncing modules fails because git/git-clone is called wrongly?

### ISSUE THREAD

**Issue by eelkevanfoeken** [kit-clj/kit#28]
https://github.com/kit-clj/kit/blob/b84d7b689115992026fb5642e78a564b0de01c28/libs/kit-generator/src/kit/generator/git.clj#L34-L38

When I attempt to do 

```
(sync-repository! "modules" {:url  "https://github.com/kit-clj/modules.git"
                             :tag  "master"
                             :name "kit-modules"})
```
this fails and gives the following error
```
failed to clone module: https://github.com/kit-clj/modules.git 
cause: No value supplied for key: {:path "modules/kit-modules", :remote-name "origin", :branch-name "master", :bare false, :clone-all-branches false}
```

When I remove the curly brackets it does work.

**Comment by yogthos** [comment]
Yeah looks like that was the issue. Thanks for the pr for the fix.

**Comment by yogthos** [comment]
Just published kit-generator 0.1.3 with the fix, looks like `:path` needed to be changed to `:dir` as well. Let me know if everything looks good on your end.
